app/services/dqt/recommend_for_award.rb

```diff
@@ -23,7 +23,7 @@ def response
     end
 
     def path
-      @path ||= "/v2/teachers/#{trainee.trn}/itt-outcome?birthDate=#{trainee.date_of_birth.iso8601}"
+      @path ||= "/v2/teachers/#{trainee.trn}/itt-outcome?slugId=#{trainee.slug}"
```

Comment:
i assume the DQT endpoint has been updated such that either `birthDate` or `slugId` can be accepted on this API version?